package.json

```diff
@@ -78,6 +78,7 @@
     "react-bootstrap": "^0.32.1",
     "react-bootstrap-typeahead": "^3.2.4",
     "react-container-dimensions": "^1.4.1",
+    "react-day-picker": "^7.3.0",
```

Comment:
do vouch for this lib?

Reply:
The library was recommended to me and it looked like it was still being maintained. I am open to using a different lib if you have one in mind

Reply:
I don't have a better choice in mind, and this one looks pretty good!  We have another day picker in netflix codebase, but we're not particularly happy with it.

Reply:
👍